__tests__/app.test.js

```diff
@@ -29,20 +29,59 @@ describe("app", () => {
           });
         });
     });
-    test("400: handles the error", () => {
-      return request(app).get("/api/notAPath").expect(404);
-      // .then((response) => {
-      //   const {
-      //     body: { msg },
-      //   } = response;
-      //   expect(msg).toBe("Not a valid path");
-      // });
+  });
+  describe("2. GET: /api/reviews/:review_id", () => {
+    test("200: responds with a review object containing properties", () => {
+      const review_id = 2;
+      return request(app)
+        .get(`/api/reviews/${review_id}`)
+        .expect(200)
+        .then((response) => {
+          const reviews = response.body.review;
```

Comment:
This represents only one review, so should be labelled to reflect it's singularity.